subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -5249,6 +5281,107 @@ static inline uint8_t phy_upd_ind_recv(struct ll_conn *conn, memq_link_t *link,
 }
 #endif /* CONFIG_BT_CTLR_PHY */
 
+#if defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
+void event_send_cis_rsp(struct ll_conn *conn)
+{
+	struct node_tx *tx;
+
+	tx = mem_acquire(&mem_conn_tx_ctrl.free);
+	if (tx) {
+		struct pdu_data *pdu = (void *)tx->pdu;
+
+		pdu->ll_id = PDU_DATA_LLID_CTRL;
+		pdu->llctrl.opcode = PDU_DATA_LLCTRL_TYPE_CIS_RSP;
+
+		sys_put_le24(conn->llcp_cis.cis_offset_min,
+			     pdu->llctrl.cis_rsp.cis_offset_min);
+		sys_put_le24(conn->llcp_cis.cis_offset_max,
+			     pdu->llctrl.cis_rsp.cis_offset_max);
+		pdu->llctrl.cis_rsp.conn_event_count =
+			sys_cpu_to_le16(conn->llcp_cis.conn_event_count);
+
+		pdu->len = offsetof(struct pdu_data_llctrl, cis_rsp) +
+				    sizeof(struct pdu_data_llctrl_cis_rsp);
+
+		conn->llcp_cis.state = LLCP_CIS_STATE_IND_WAIT;
+
+		ctrl_tx_enqueue(conn, tx);
+	}
+}
+
+void event_peripheral_iso_prep(struct ll_conn *conn, uint16_t event_counter,
+			       uint32_t ticks_at_expire)
+{
+	if (event_counter == conn->llcp_cis.conn_event_count) {
+		ull_peripheral_iso_start(conn, ticks_at_expire);
+
+		conn->llcp_cis.state = LLCP_CIS_STATE_REQ;
+		conn->llcp_cis.req = conn->llcp_cis.ack;
```

Comment:
```suggestion
		conn->llcp_cis.ack = conn->llcp_cis.req;
```

See my comment regarding use of `req`.